src/backend/dwarf.c

```diff
@@ -790,6 +790,20 @@ void writeEhFrameFDE(IDXSEC dfseg, Symbol *sfunc)
     Outbuffer *buf = SegData[dfseg]->SDbuf;
     const unsigned startsize = buf->size();
 
+    if (sfunc->ty() & mTYnaked)
+    {
+        /* Do not have info on naked functions. Assume they are set up as:
+         *   push RBP
+         *   mov  RSP,RSP
+         */
+        int off = 2 * REGSIZE;
+        dwarf_CFA_set_loc(1);
+        dwarf_CFA_set_reg_offset(SP, off);
+        dwarf_CFA_offset(BP, -off);
+        dwarf_CFA_set_loc(I64 ? 4 : 3);
+        dwarf_CFA_set_reg_offset(BP, off);
+    }
+
```

Comment:
> The stack frame info is necessary only if the inline assembler does not call any functions that throw. The etc.linux.memoryerror does do this. I considered removing memoryerror, as it converts seg faults into thrown exceptions, something that doesn't belong in a systems programming language, but decided to simply make it work for now and leave that issue for a separate day.

I tried hard to prevent memoryerror from happening but couldn't achieve more than moving it to etc. In any case people are now relying on it to get tracebacks, e.g. for server apps, so w/o an alternative it would be very rude to remove it.